README.md

```diff
@@ -1,8 +1,16 @@
-# Installation
-                          
-> ⚠️ Please use the Grafana Agent configuration for production use cases. 
+# Overview
+
+The grafana-opentelemetry-starter makes it easy to use Metrics, Traces, and Logs with OpenTelemetry 
+in Grafana Cloud or with Grafana Agent (for Grafana Cloud or Grafana OSS stack). 
+
+# Compatibility
 
-Spring boot starter to use Metrics, Traces, and Logs with OpenTelemetry in Grafana Cloud or with Grafana Agent (for Grafana Cloud or Grafana OSS stack) 
+| Spring Boot Version | Recommended Setup                                                                        |
+|---------------------|------------------------------------------------------------------------------------------|
+| 3.0.4+              | Use this starter                                                                         |
```

Comment:
Does it make sense to add a column indicating Java compatibility? 

```
3.x: Java 17+
2.x: Java 8+
```

^ I believe that's correct.